src/main/resources/music/settings/host/values/strings.xml

```diff
@@ -15,63 +15,63 @@
-    <string name="revanced_enable_color_match_player_summary">Matches the color of the mini player and the fullscreen player.</string>
+    <string name="revanced_enable_color_match_player_summary">Matches the color of the miniplayer to the fullscreen player.</string>
     <string name="revanced_enable_color_match_player_title">Enable color match player</string>
-    <string name="revanced_enable_compact_dialog_summary">"Enable compact dialog on phone.
+    <string name="revanced_enable_compact_dialog_summary">"Enables the compact flyout menu on phones.
 
 Known issues:
-• Album art on the library shelf also becomes smaller.
+• Album art in the library tab becomes smaller when organized in a grid.
 • Sleep timer layout may appear unusual."</string>
     <string name="revanced_enable_compact_dialog_title">Enable compact dialog</string>
     <string name="revanced_enable_debug_logging_summary">Prints the debug log.</string>
     <string name="revanced_enable_debug_logging_title">Enable debug logging</string>
-    <string name="revanced_enable_flyout_panel_playback_speed_summary">Add an playback speed button to the flyout panel.</string>
+    <string name="revanced_enable_flyout_panel_playback_speed_summary">Add a playback speed button to the flyout panel.</string>
     <string name="revanced_enable_flyout_panel_playback_speed_title">Enable playback speed</string>
-    <string name="revanced_enable_force_minimized_player_summary">Keep player permanently minimized even if another track is played.</string>
+    <string name="revanced_enable_force_minimized_player_summary">Keeps the miniplayer minimized even when another track is played.</string>
     <string name="revanced_enable_force_minimized_player_title">Enable force minimized player</string>
-    <string name="revanced_enable_landscape_mode_summary">Enables entry into landscape mode by screen rotation on the phone.</string>
+    <string name="revanced_enable_landscape_mode_summary">Enables landscape mode when rotating the screen on phones.</string>
     <string name="revanced_enable_landscape_mode_title">Enable landscape mode</string>
-    <string name="revanced_enable_new_player_background_summary">Enables new player background.</string>
+    <string name="revanced_enable_new_player_background_summary">Enables the new player background.</string>
     <string name="revanced_enable_new_player_background_title">Enable new player background</string>
-    <string name="revanced_enable_old_player_layout_summary">"Return the player layout to old style.
-Some settings may not work properly in the old player layout.
+    <string name="revanced_enable_old_player_layout_summary">"Returns the player layout to the old style.
+Some features may not work properly in the old player layout.
 
-WARNING: Do not enable new player backgrounds while this is enabled."</string>
+WARNING: Do not enable \"New player background\" while this is enabled."</string>
```

Comment:
Instead of

```
WARNING: Do not enable \"New player background\" while this is enabled.
```

It must be

```
WARNING: Do not enable "New player background" while this is enabled.
```

According to XML's grammar, `revanced_enable_old_player_layout_summary` is surrounded by a large quotes as follows:

```
    <string name="revanced_enable_old_player_layout_summary">"Returns the player layout to the old style.
Some features may not work properly in the old player layout.

WARNING: Do not enable \"New player background\" while this is enabled."</string>
```

If you are surrounded by such a large quotes, you should not use escape sequences `\`

See the documentation for XML's grammar for more information.

[Google's doc](https://developer.android.com/guide/topics/resources/string-resource?hl=en#escaping_quotes)
[Microsoft's doc](https://learn.microsoft.com/en-us/dotnet/desktop/xaml-services/xml-character-entities#xml-character-entities)